src/renderers/dom/stack/client/wrappers/ReactDOMInput.js

```diff
@@ -122,6 +123,7 @@ var ReactDOMInput = {
       initialChecked: props.checked != null ? props.checked : props.defaultChecked,
       initialValue: props.value != null ? props.value : defaultValue,
       listeners: null,
+      onBlur: _handleBlur.bind(inst),
```

Comment:
We need to make sure this is invoked when the input is blurred, and add a test if possible.